run-tests.sh

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+cd "$(dirname "$0")"
+
+port=9000
+containers=()
+network=$(LC_ALL=C tr -dc 'a-z' </dev/urandom | head -c 13 ; echo)
```

Comment:
this way of generating a random name is used twice in the script

I think it would be worth putting in a function

Reply:
👍